app/controllers/assessment360_controller.rb

```diff
@@ -15,21 +15,19 @@ def all_students_all_reviews
     course = Course.find(params[:course_id])
     @assignments = course.assignments.reject(&:is_calibrated).reject {|a| a.participants.empty? }
     @course_participants = course.get_participants
-    if @course_participants.empty?
-      flash[:error] = "There is no course participant in course #{course.name}"
-      redirect_to(:back)
-    end
+    inspect_course_participants(@course_participants)
```

Comment:
It is not clear from the name of this method what it does.  How about, "insure_existence_of"?